src/content/en/2019/accessibility.md

```diff
@@ -245,40 +247,43 @@ The relative popularity of the [dialog role](https://developer.mozilla.org/en-US
 
 #### Labels on interactive elements
 
-The most common way that a user interacts with a website is through its controls, such as links or buttons, in order to perform an action such as navigating to another part of the website, or changing what is displayed on the current page. Many times, screen reader users are unable to tell what action a control, such as a button or a link, is going to perform once activated. The reason is often the fact that the control lacks a textual label. For example, it might convey its meaning only in a graphical way, such as a button displaying a left-pointing arrow to signify that it is the "Back" button, or showing an “X” to signify that it is the “Close” button.
+The most common way that a interacts with a website is through its controls (such as links or buttons to navigate the website). However, many times screen reader users are unable to tell what action a control will perform once activated. Often the reason this confusion occurs is due to the lack of a textual label. For example: a button displaying a left-pointing arrow icon to signify it's the "Back" button, but containing no actual text.
 
-In our sample, a lot of websites use buttons or links. From those, **only around a quarter, or 24.39%,** include textual labels with these controls. If a control is not labeled, a screen reader user might read something generic, such as the word "button," instead of a meaningful name such as “Search”. This makes it difficult for screen reader users to understand how to use the website.
+In our sample, a lot of websites use buttons or links. From those, only around a quarter (24.39%) include textual labels with these controls. If a control is not labeled a screen reader user might read something generic, such as the word "button" instead of a meaningful word like "Search".
```

Comment:
```suggestion
Only about a quarter (24.39%) of pages that use buttons or links include textual labels with these controls. If a control is not labeled, a screen reader user might read something generic, such as the word "button" instead of a meaningful word like "Search".
```